scout/load/setup.py

```diff
@@ -65,7 +65,7 @@ def setup_scout(
 
     LOG.info("Check if there was a database, delete if existing")
     existing_database = False
-    for collection_name in adapter.db.collection_names():
+    for collection_name in adapter.db.list_collection_names():
```

Comment:
Why not use the `adapter.collections` function that you changed above? Next time we would only need to update in one place :)